src/sentry/integrations/msteams/client.py

```diff
@@ -17,7 +17,11 @@
 class MsTeamsAbstractClient(ApiClient):
     integration_name = "msteams"
     TEAM_URL = "/v3/teams/%s"
+    CHANNEL_URL = "/v3/teams/%s/conversations"
     ACTIVITY_URL = "/v3/conversations/%s/activities"
+    CONVERSATION_URL = "/v3/conversations"
+    MEMBER_URL = "/v3/conversations/%s/pagedmembers?pageSize=500"
```

Comment:
@KKaempen Instead of putting the `pageSize` as part of the URL, it would be better to specify it in `params `.

Reply:
Yeah, good call. I'll do the same for the continuation token.